<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Deduplicate</class>
 <widget class="QWidget" name="Deduplicate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>185</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:transparent;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QCheckBox" name="m_pAddColumn">
     <property name="text">
      <string>Add column</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Threshold:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="m_pThreshold"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Column:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>31</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_pAddColumn</sender>
   <signal>toggled(bool)</signal>
   <receiver>Deduplicate</receiver>
   <slot>slotEnableExtraColumn(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>22</x>
     <y>8</y>
    </hint>
    <hint type="destinationlabel">
     <x>12</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pThreshold</sender>
   <signal>valueChanged(double)</signal>
   <receiver>Deduplicate</receiver>
   <slot>slotChangeTreshold(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>59</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Deduplicate</receiver>
   <slot>slotIndexChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotEnableExtraColumn(bool)</slot>
  <slot>slotChangeTreshold(double)</slot>
  <slot>slotIndexChanged(int)</slot>
 </slots>
</ui>
